assets/dbs/abilities_db.json

```diff
@@ -2087,5 +2087,308 @@
+        "type": "effect_only",
+        "element": "jupiter",
+        "battle_target": "enemy",
+        "range": 3,
+        "pp_cost": 5,
+        "ability_power": 0,
+        "effects_outside_battle": false,
+        "is_battle_ability": true,
+        "is_field_psynergy": false,
+        "effects": [
+            {
+                "type": "temporary_status",
+                "chance": 0.1,
+                "add_status": true,
+                "status_key_name": "sleep",
+                "usage": "on_use",
+                "turns_quantity": 7
+            }
+        ],
+        "ability_category": "psynergy",
+        "battle_animation_key": "no_animation",
+        "priority_move": false,
+        "msg_type": "cast"
+    },{
+        "key_name": "heat_juggle",
+        "name": "Heat Juggle",
+        "description": "Skillfully toss balls of flame.",
+        "type": "base_damage",
+        "element": "mars",
+        "battle_target": "enemy",
+        "range": 5,
+        "pp_cost": 13,
+        "ability_power": 75,
+        "effects_outside_battle": false,
+        "is_battle_ability": true,
+        "is_field_psynergy": false,
+        "effects": [],
+        "ability_category": "psynergy",
+        "battle_animation_key": "no_animation",
+        "priority_move": false,
+        "msg_type": "cast"
+    },{
+        "key_name": "fire_breath",
+        "name": "Fire Breath",
+        "description": "Attack with a sheet of flames.",
+        "type": "base_damage",
+        "element": "mars",
+        "battle_target": "enemy",
+        "range": 3,
+        "pp_cost": 13,
+        "ability_power": 85,
+        "effects_outside_battle": false,
+        "is_battle_ability": true,
+        "is_field_psynergy": false,
+        "effects": [],
+        "ability_category": "psynergy",
+        "battle_animation_key": "no_animation",
+        "priority_move": false,
+        "msg_type": "cast"
+    },{
+        "key_name": "thunder_card",
+        "name": "Thunder Card",
+        "description": "Throw a card of the Thunder suit.",
+        "type": "base_damage",
+        "element": "jupiter",
+        "battle_target": "enemy",
+        "range": 3,
+        "pp_cost": 17,
+        "ability_power": 100,
+        "effects_outside_battle": false,
+        "is_battle_ability": true,
+        "is_field_psynergy": false,
+        "effects": [],
+        "ability_category": "psynergy",
+        "battle_animation_key": "no_animation",
+        "priority_move": false,
+        "msg_type": "cast"
+    },{
+        "key_name": "death_card",
+        "name": "Death Card",
+        "description": "Call the Reaper to claim your foes.",
+        "type": "effect_only",
+        "element": "venus",
+        "battle_target": "enemy",
+        "range": 1,
+        "pp_cost": 8,
+        "ability_power": 0,
+        "effects_outside_battle": false,
+        "is_battle_ability": true,
+        "is_field_psynergy": false,
+        "effects": [
```

Comment:
We need the 0.2 chance here.
![image](https://user-images.githubusercontent.com/7890647/103484429-92b67e80-4dcd-11eb-802b-c79b3f307625.png)